ui/ducks/metamask/metamask.js

```diff
@@ -344,10 +346,36 @@ export function getGasEstimateType(state) {
   return state.metamask.gasEstimateType;
 }
 
-export function getGasFeeEstimates(state) {
+export function getGasFeeControllerEstimates(state) {
   return state.metamask.gasFeeEstimates;
 }
 
+export function getTransactionGasFeeEstimates(state) {
```

Comment:
Matt, should we name this function getConfirmTransactionGasFeeEstimates? I'm wondering if it's clear enough from the current name that it's related to confirmTransaction.

Reply:
I believe so, the concept of a pending transaction is fundamental to the UI, and the related reducer is likely called `confirmTransaction` just to differentiate it from the `TransactionController` state.

But for this new selector, the caller doesn't care about the specific reducer involved, only that it returns the estimates specific to the transaction.